.envrc

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
why? never seen this in a `.envrc`

Reply:
I intended to remove and forgot. gone now